### PR TITLE
Add in variables that are likely to have been configured by the user

### PR DESCRIPTION
**Pull request by hshin-g**
Add in variables that are likely to have been configured by the user for better clarity.

### CODE REVIEW

File: examples/migrate_forseti/tutorial.md [hunk 1]
```diff
@@ -149,21 +149,43 @@ Starting with Forseti Security 2.23, Terraform will manage your server
 Please identify any Forseti server configuration variables that have
 been customized and add them to your <walkthrough-editor-open-file filePath="terraform-google-forseti/examples/migrate_forseti/main.tf">main.tf</walkthrough-editor-open-file>.
 
-For example, if you modified the e-mail connector recipient in the **forseti_conf_server.yaml** file in the Forseti Server GCS bucket:
+The default values for the variables can be found 
+[here](https://github.com/forseti-security/terraform-google-forseti/blob/module-release-5.0.0/variables.tf), and will 
+be set to the default if they are not explicitly added.
+
+The following variables have been listed to help you identify and set any customized values:
```
Comment (blueandgold) [inline examples/migrate_forseti/tutorial.md]: Maybe we should phrase this as:

```
The following variables have been listed as a sample to help you identify and set any customized values.  There maybe other variables with customized values that will need to be set.
```

File: examples/migrate_forseti/tutorial.md [hunk 2]
```diff
@@ -149,21 +149,43 @@ Starting with Forseti Security 2.23, Terraform will manage your server
 Please identify any Forseti server configuration variables that have
```
Comment (blueandgold) [inline examples/migrate_forseti/tutorial.md]: Please prefix this with `IMPORTANT: `

Comment (hshin-g) [reply examples/migrate_forseti/tutorial.md]: Addressed- thanks!

File: examples/migrate_forseti/tutorial.md (hunk 1 above)
Comment (hshin-g) [reply examples/migrate_forseti/tutorial.md]: Added in- thanks!

**Pull request merged**